webapps/frontend/lib/ovs/api.js

```diff
@@ -34,7 +34,7 @@ define([
 
         callData = {
             type: type,
-            timeout: generic.tryGet(options, 'timeout', 1000 * 30),
+            timeout: generic.tryGet(options, 'timeout', 1000 * 120),
```

Comment:
I'd suggest a little longer than 120s. The arakoon calls can take up to 120s to timeout alone, leave some room for additional computing time.